website/templates/log_list.mako

```diff
@@ -46,12 +46,17 @@
                                 <span class="contributor-anonymous">A user</span>
                             </span>
                                 <span data-bind="ifnot:log.anonymous">
+                                    <span data-bind="if: log.user.registered">
```

Comment:
Logs on the "My projects" page are rendered with separate code than the logs on project Overview pages. See `logTextParser.js` and `logActionsList.json`. You may need to make a corresponding change there to make the logs display consistently.